diff --git a/Makefile b/Makefile
index 4692cd5..037d738 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,10 @@
 # makefile for srlua
 
 # change these to reflect your Lua installation
-LUA= /tmp/lhf/lua-5.2.0
-LUAINC= $(LUA)/src
-LUALIB= $(LUA)/src
-LUABIN= $(LUA)/src
+LUA= ../.lua
+LUAINC= $(LUA)/include
+LUALIB= $(LUA)/lib
+LUABIN= $(LUA)/bin
 
 # these will probably work if Lua has been installed globally
 #LUA= /usr/local
@@ -17,12 +17,12 @@ CC= gcc
 CFLAGS= $(INCS) $(WARN) -O2 $G
 WARN= -ansi -pedantic -Wall -Wextra
 INCS= -I$(LUAINC)
-LIBS= -L$(LUALIB) -llua -lm -ldl
+LIBS= -L$(LUALIB) -llua54 -lm -ldl
 EXPORT= -Wl,-E
 # for Mac OS X comment the previous line above or do 'make EXPORT='
 
 T= a.out
-S= srlua
+S= srlua.exe
 OBJS= srlua.o
 TEST= test.lua
 
