diff --git a/Makefile b/Makefile
index 4692cd5..f2eda60 100644
--- a/Makefile
+++ b/Makefile
@@ -1,16 +1,10 @@
 # makefile for srlua
 
 # change these to reflect your Lua installation
-LUA= /tmp/lhf/lua-5.2.0
-LUAINC= $(LUA)/src
-LUALIB= $(LUA)/src
-LUABIN= $(LUA)/src
-
-# these will probably work if Lua has been installed globally
-#LUA= /usr/local
-#LUAINC= $(LUA)/include
-#LUALIB= $(LUA)/lib
-#LUABIN= $(LUA)/bin
+LUA= $(GITHUB_WORKSPACE)/.lua
+LUAINC= $(LUA)/include
+LUALIB= $(LUA)/lib
+LUABIN= $(LUA)/bin
 
 # probably no need to change anything below here
 CC= gcc
@@ -22,7 +16,7 @@ EXPORT= -Wl,-E
 # for Mac OS X comment the previous line above or do 'make EXPORT='
 
 T= a.out
-S= srlua
+S= srlua.exe
 OBJS= srlua.o
 TEST= test.lua
 
@@ -31,14 +25,13 @@ all:	test
 test:	$T
 	./$T *
 
-$T:	$S $(TEST) glue
-	./glue $S $(TEST) $T
-	chmod +x $T
+$T:	$S $(TEST) glue.exe
+	./glue.exe $S $(TEST) $T
 
 $S:	$(OBJS)
 	$(CC) -o $@ $(EXPORT) $(OBJS) $(LIBS)
 
 clean:
-	rm -f $(OBJS) $T $S core core.* a.out *.o glue
+	rm -f $(OBJS) $T $S core core.* a.out *.o glue.exe
 
 # eof
