diff --git a/Makefile b/Makefile
index 4692cd5..193780b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,28 +1,22 @@
 # makefile for srlua
 
 # change these to reflect your Lua installation
-LUA= /tmp/lhf/lua-5.2.0
-LUAINC= $(LUA)/src
-LUALIB= $(LUA)/src
-LUABIN= $(LUA)/src
-
-# these will probably work if Lua has been installed globally
-#LUA= /usr/local
-#LUAINC= $(LUA)/include
-#LUALIB= $(LUA)/lib
-#LUABIN= $(LUA)/bin
+LUA= $(GITHUB_WORKSPACE)/.lua
+LUAINC= $(LUA)/include
+LUALIB= $(LUA)/lib
+LUABIN= $(LUA)/bin
 
 # probably no need to change anything below here
 CC= gcc
 CFLAGS= $(INCS) $(WARN) -O2 $G
 WARN= -ansi -pedantic -Wall -Wextra
 INCS= -I$(LUAINC)
-LIBS= -L$(LUALIB) -llua -lm -ldl
-EXPORT= -Wl,-E
+LIBS= -L$(LUALIB) -llua51 -lm
+EXPORT= -Wl, --export-all-symbols
 # for Mac OS X comment the previous line above or do 'make EXPORT='
 
 T= a.out
-S= srlua
+S= srlua.exe
 OBJS= srlua.o
 TEST= test.lua
 
@@ -31,14 +25,16 @@ all:	test
 test:	$T
 	./$T *
 
-$T:	$S $(TEST) glue
-	./glue $S $(TEST) $T
-	chmod +x $T
+$T:	$S $(TEST) glue.exe
+	./glue.exe $S $(TEST) $T
 
 $S:	$(OBJS)
 	$(CC) -o $@ $(EXPORT) $(OBJS) $(LIBS)
 
+glue.exe: glue.o
+	$(CC) -o $@ glue.o $(EXPORT) $(LIBS)
+
 clean:
-	rm -f $(OBJS) $T $S core core.* a.out *.o glue
+	rm -f $(OBJS) $T $S core core.* a.out *.o glue.exe
 
 # eof
diff --git a/glue.c b/glue.c
index d7c81c0..7b833e1 100644
--- a/glue.c
+++ b/glue.c
@@ -14,6 +14,7 @@
 
 static void cannot(const char* what, const char* name)
 {
+ printf("oh no!\n");
  fprintf(stderr,"cannot %s %s: %s\n",what,name,strerror(errno));
  exit(EXIT_FAILURE);
 }
